<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContractForm</class>
 <widget class="QWidget" name="ContractForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditExchange">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>symbol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSymbol">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>exchange</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonTick">
       <property name="text">
        <string>showTick</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>s01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s05</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m05</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h02</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h04</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>w01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x01</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBar">
       <property name="text">
        <string>showBar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
